src/views/Login.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import http from 'superagent'
+
+export class Login extends React.Component {
+
+  constructor () {
+    super()
+    this.state = {
+      user: '',
+      password: ''
+    }
+  }
+
+  setUser = e => {
```

Comment:
linter was complaining about using `this.setUser.bind(this)` in the render function, so after a little research, found that `=>` binds the function to the scope of `this`. is there a better pattern for this?